templates/pioneer-citp/news-disinfo-study/news-disinfo-study.1.schema.json

```diff
@@ -0,0 +1,86 @@
+{
+    "mozPipelineMetadata": {
+        "bq_metadata_format": "pioneer"
+    },
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object", "properties": {
+        "WebScience.Measurements.PageNavigation": { "type": "object", "properties": {
+            "numUntrackedVisits": {"type": "number"},
+            "trackedVisitsByDomain": {"type": "array",
+                "items": { "type": "object", "properties": {
+                    "domain" : {"type": "string"},
+                    "numSessions": {"type": "number"},
+                    "visitsByReferrer": {"type": "array",
+                        "items": {"type": "object", "properties": {
+                            "referrerDomain": {"type": "string"},
+                            "dayOfWeek": {"type": "number",
+                                "minimum": 0, "maximum": 6},
+                            "timeOfDay": {"type": "number",
+                                "multipleOf": 4, "minimum": 0, "maximum": 20},
```

Comment:
I'm not sure how strict you want to be with validation here, a single validation error will cause the whole ping to be rejected. It's fairly straightforward to filter data in SQL and not have to worry about small bugs causing dropped pings. If this schema is the same one applied client-side, it should be all good.

Reply:
Yeah, this is something I've been wondering about: the line between "appropriate validating" and "breaking everything on one tiny bug" seems potentially fuzzy. I'll look back through the schemas and code to find a reasonable place to land.